go.mod

```diff
@@ -118,6 +120,7 @@ require (
 	github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24 // indirect
 	github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94 // indirect
 	github.com/stretchr/testify v1.3.0
+	github.com/y0ssar1an/q v1.0.7
```

Comment:
Run `go mod tidy`

Reply:
You saw nothing ... 


(done in 7a3db9941)